contrib/opentelemetry/tracing_interceptor.go

```diff
@@ -86,6 +86,14 @@ type TracerOptions struct {
 	// SpanStarter is a callback to create spans. If not set, this creates normal
 	// OpenTelemetry spans calling Tracer.Start.
 	SpanStarter func(ctx context.Context, t trace.Tracer, spanName string, opts ...trace.SpanStartOption) trace.Span
+
+	// TraceIDFieldName is the name of the field used to represent the Trace ID
+	// If empty, a default name will be used.
+	TraceIDFieldName string
+
+	// SpanIDFieldName is the name of the field used to represent the Span ID
+	// If empty, a default name will be used.
+	SpanIDFieldName string
```

Comment:
```suggestion
	// TraceIDLoggerFieldName is the name of the field used to represent the Trace ID in logger.
	// If empty, a default name will be used.
	TraceIDLoggerFieldName string

	// SpanIDLoggerFieldName is the name of the field used to represent the Span ID in logger.
	// If empty, a default name will be used.
	SpanIDLoggerFieldName string
```

Not clear this is only for logger and nothing else